packages/component-library/src/components/plmg-text-input/plmg-text-input.tsx

```diff
@@ -195,17 +408,50 @@ export class TextInput {
           {this.showLabel && this.required && <span class={'required'}>*</span>}
         </label>
         <div class={'plmg-text-input-field-wrapper'} tabIndex={0}>
-          <input
-            disabled={this.disabled}
-            class={inputClasses}
-            id={this.labelToId()}
-            name={this.label}
-            required={this.required}
-            type={'text'}
-            readonly={this.readOnly}
-            value={this.internalValue}
-            onInput={(ev) => this.handleInputChange(ev)}
-          />
+          {this.type === 'number' ? (
+            <input
+              autoComplete={this.autoComplete}
+              class={inputClasses}
+              disabled={this.disabled}
+              id={this.labelToId()}
+              max={this.max}
+              min={this.min}
+              name={this.name}
+              onInput={(event) => this.handleInputChange(event.target)}
+              placeholder={this.placeholder}
+              readonly={this.readOnly}
+              required={this.required}
+              size={this.width}
+              step={this.step}
+              style={{
```

Comment:
missing from the other case

Reply:
Explanation above. I was using size to set width with non-number fields